Sources/SwiftInspectorVisitors/Tests/ClassVisitorSpec.swift

```diff
@@ -1,413 +0,0 @@
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass.BarFooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.FooFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "FooFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.FooFooClass.BarFooFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "BarFooFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass.FooFooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.BarFooEnum") {
-            let matching = self.sut.innerEnums.filter {
-              $0.name == "BarFooEnum"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.BarFooEnum.BarBarFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "BarBarFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass.BarFooEnum"
-            }
-
-            expect(matching.count) == 1
-          }
-        }
-      }
-
-      context("visiting a code block with multiple top-level declarations") {
-        context("with multiple top-level classes") {
-          it("asserts") {
-            let content = """
-            public class FooClass {}
-            public class BarClass {}
-            """
-
-            // The ClassVisitor is only meant to be used over a single class.
-            // Using a ClassVisitor over a block that has multiple top-level
-            // classes is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level struct after a top-level class") {
-          it("asserts") {
-            let content = """
-            public class FooClass {}
-            public struct FooStruct {}
-            """
-
-            // The ClassVisitor is only meant to be used over a single class.
-            // Using a ClassVisitor over a block that has a top-level struct
-            // is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level enum after a top-level class") {
```

Comment:
This context was brought into the new file under a context named `with a top-level enum declaration`